smt/applications/ego.py

```diff
@@ -71,6 +71,13 @@ def _initialize(self):
             types=int,
             desc="Maximum number of internal optimizations",
         )
+        # New declaration for noise0
+        declare(
+            "noise0",
```

Comment:
I do not see any usage of options["noise0"]. We do not need to declare a `noise0` option in EGO, it is already an option of `KRG`. To be removed.